build.gradle

```diff
@@ -0,0 +1,19 @@
+plugins {
+    id 'io.franzbecker.gradle-lombok' version '1.11'
```

Comment:
Why do you need plugins for lombok? No other project has them, right?

Reply:
It might be useful for delombokking when publishing, so we only have to add lombok as an dev dependency :)

Reply:
https://projectlombok.org/setup/gradle